spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/actions/DeleteReachableFilesSparkAction.java

```diff
@@ -54,6 +55,8 @@
   private static final Logger LOG = LoggerFactory.getLogger(DeleteReachableFilesSparkAction.class);
 
   private final String metadataFileLocation;
+
+  @Deprecated
   private final Consumer<String> defaultDelete =
```

Comment:
The code still uses this var even though it is marked as deprecated. Can we simply remove it?